Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Active_Directory_Enrichment.yml

```diff
@@ -0,0 +1,451 @@
+      id: 70e25786-8973-4ba9-81f2-cb67bebc3272
+      iscommand: true
+      name: Lookup user details in Active Directory.
+      script: Microsoft Graph User|||msgraph-user-get
+      type: regular
+      version: -1
+    taskid: 70e25786-8973-4ba9-81f2-cb67bebc3272
+    timertriggers: []
+    type: regular
+    view: |-
+      {
+        "position": {
+          "x": 500,
+          "y": 370
+        }
+      }
+  "8":
+    conditions:
+    - condition:
+      - - left:
+            iscontext: true
+            value:
+              complex:
+                filters:
+                - - left:
+                      iscontext: true
+                      value:
+                        simple: modules.state
+                    operator: isEqualString
+                    right:
+                      value:
+                        simple: active
+                - - left:
+                      iscontext: true
+                      value:
+                        simple: modules.brand
+                    operator: isEqualString
+                    right:
+                      value:
+                        simple: Microsoft Graph User
+                root: modules
+          operator: isExists
+          right:
+            value: {}
+      - - left:
+            iscontext: true
+            value:
+              simple: inputs.serviceowner
+          operator: isNotEmpty
+      label: "yes"
+    continueonerrortype: ""
+    id: "8"
+    ignoreworker: false
+    isautoswitchedtoquietmode: false
+    isoversize: false
+    nexttasks:
+      '#default#':
+      - "2"
+      "yes":
+      - "7"
+    note: false
+    quietmode: 0
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: ""
+      description: Determines if the Microsoft Graph User integration instance is configured.
+      id: b21e2b1d-6220-47d6-81a2-fdb941916747
+      iscommand: false
+      name: Is Microsoft Graph User enabled and Input defined?
+      type: condition
+      version: -1
+    taskid: b21e2b1d-6220-47d6-81a2-fdb941916747
+    timertriggers: []
+    type: condition
+    view: |-
+      {
+        "position": {
+          "x": 162.5,
+          "y": 195
+        }
+      }
+  "10":
+    continueonerrortype: ""
+    id: "10"
+    ignoreworker: false
+    isautoswitchedtoquietmode: false
+    isoversize: false
+    nexttasks:
+      '#none#':
+      - "14"
+    note: false
+    quietmode: 0
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: ""
+      id: f16dc8b2-5f06-4194-8ad0-715d6cb4151c
+      iscommand: false
+      name: Set Fields
+      type: title
+      version: -1
+      description: ''
+    taskid: f16dc8b2-5f06-4194-8ad0-715d6cb4151c
+    timertriggers: []
+    type: title
+    view: |-
+      {
+        "position": {
+          "x": 500,
+          "y": 535
+        }
+      }
+  "11":
```

Comment:
I think we should avoid the branching to set values to `asmserviceownerunrankedraw` if possible. Would something like this work for each value:

![image](https://github.com/demisto/content/assets/40349459/a4b1816d-4f50-4d15-9f29-fee28c11de72)